portal/server/routes/contract/bond/index.js

```diff
@@ -57,6 +58,36 @@ router.get('/contract/:_id/bond/create', async (req, res) => {
   return res.redirect(`/contract/${dealId}/bond/${bondId}/details`);
 });
 
+const saveBondAndGoBackToDeal = async (req, res, sanitizedBody) => {
```

Comment:
Should this be a helper since same is being repeated for `loan`.

Reply:
It's almost the same with a  different API call, what me to combine them and use a flag to determine which API call to use?

Reply:
Agreed for help function insertion to mitigate `DRY`